tests/src/test/scala/org/apache/openwhisk/core/scheduler/container/test/ContainerManagerTests.scala

```diff
@@ -910,6 +910,49 @@ class ContainerManagerTests
         NoAvailableInvokersError))
   }
 
+  it should "send FailedCreationJob to queue manager when available invoker query fails" in {
```

Comment:
Seems the title should be changed from `to queue manager` to `memory queue`
https://github.com/apache/openwhisk/blob/master/core/scheduler/src/main/scala/org/apache/openwhisk/core/scheduler/container/ContainerManager.scala#L580